### ISSUE TITLE
FP for oelint.vars.specific for bbappend

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#198]
following snippet for an openssl bbappend
```
FILES_libcrypto += "${libdir}/libcrypto.so.1"
```
gives the false positives
`Variable FILES is package-specific and therefore it should be FILES_${PN}`
and
`'FILES' is set specific to ['libcrypto'], but isn't known from PACKAGES, MACHINE or resources`

 concluding, if a bb to bbappend if not known, it does make sense to post these warnings
